### PR TITLE
Init Project

### PR DESCRIPTION
**Pull request by lrmi0808**
Redone Init project due to confusing problems. Ready.

### CODE REVIEW

**Comment by lrmi0808** [comment]
Ready.

**Comment by lrmi0808** [comment]
Ready.

File: config/database.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# SQLite. Versions 3.8.0 and up are supported.
+#   gem install sqlite3
+#
+#   Ensure the SQLite 3 gem is defined in your Gemfile
+#   gem 'sqlite3'
+#
+default: &default
```
Comment (ngocvt-0484) [inline config/database.yml:7]: this file will not be up to GIt, please
```
git rm --cached config/database.yml
```
and in .gitignore add
```
config/database.yml
```

**Comment by lrmi0808** [comment]
Ready

**Comment by ngocvt-0484** [comment]
merged

**Pull request merged**